docs/RFCS/20220608_split_jobs_system_tables.md

```diff
@@ -0,0 +1,279 @@
+
+
+    	# claim/backoff fields determine who should/when to try execute this job
+    	claim_session_id BYTES,
+    claim_instance_id INT,
+    	backoff_num_attempts INT, 
+    	backoff_last_attempt TIMESTAMPTZ,
+
+    	# families: split out zero-churn large text and high-churn small numbers.
+    	FAMILY immutable (job_type, description, created, created_by_type, created_by_id),
+
+    	# indexes
+    INDEX by_state (state, created),
+    	INDEX by_type (job_type, state, created_by_type, created_by_id),
+    	INDEX claimable (claim_session_id, state, created) STORING (backoff_last_attempt, backoff_num_attempts, claim_instance_id)
+    );
+```
+
+#### Job Control Statements and Their Transactions
+
+User-facing job control statements will result in transactions in this table, such as to update a running job's status to pause-requested. This matches the current behavior with respect to writing a pause-request to the current system.jobs, however the goal of eliminating all blocking transactions from the new control table suggests that such control statements should be updated to no longer be allowed in explicit, user-owned transactions. In many ways, this added restriction should have been there already: running `PAUSE JOB X` inside a transaction can be confusing, because it does not actually pause the running job until it is committed, at which time a pause-request is written, then at some later time observed, and only then does the job itself move to paused. However if adding this new restriction proves unpalatable, options include either simply allowing these blocking transactions and the fact they block the jobs system --  if a user has indicated they will pause a job they need to either commit that fact or abort it before we can answer if the job is paused or not -- or we can introduce additional queued operation tables similar to the intake table described below. 
+
+
+###### Upgrade Path for system.jobs
+
+Changes to the job control state, as jobs step through the control state machine, would need to start writing to both tables, along with job creation (writing to intake). A migration can then copy all existing jobs to the new control table in the appropriate state to backfill. Subsequently control queries can begin to query the new table instead, until an additional cluster version disables writes to the existing table, followed by a migration that drops it. Note that writes to job detail and progress are discussed in the job_info section below; this would be for control-related writes only.
+
+---
+
+### system.job_status_progress + system.job_status_message
+
+#### Description
+
+The job_status_progress and system.job_status_messages tables store human-readable information about a job's current and recent status and progress.
+
+These tables are not read by the jobs control system at any point, and should not be read by the job itself either; they are intended solely for human-consumption via SHOW JOBS and the DB Console. 
```

Comment:
when a job's status changes, will the job system update the control_table in the same txn as it inserts a new row in the job_status_message table? Or is the status table allowed to lag behind the control table?